src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -161,11 +161,14 @@ public bool PreAuthenticate
             get { return true; }
             set
             {
+                /*
+                TODO:#18104
```

Comment:
What's the value of commenting this out? If code is depending on preauthenticate working,  it'll still be broken, just in a different way... and at least with the PNSE it's very obvious it's broken.